cmd/status/printers/event/event_printer.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2020 The Kubernetes Authors.
```

Comment:
I recognize this practice already exists, but it feels a little odd to place files in subdirectories of the `cmd/foo`. We can discuss this file organization later.